content/ja/docs/getting-started/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Getting Started
```

Comment:
Should the title be translated too?

Reply:
Even in Japanese tech docs, it's common to have "getting started" as-is, but I put additional translation just in case.